packages/ckeditor5-fullscreen/src/fullscreenediting.ts

```diff
@@ -43,5 +45,21 @@ export default class FullscreenEditing extends Plugin {
 	 */
 	public init(): void {
 		this.editor.commands.add( 'fullscreen', new FullscreenCommand( this.editor ) );
+
+		const t = this.editor.locale.t;
+
+		// Set the Ctrl+Shift+F keystroke.
+		this.editor.keystrokes.set( 'Ctrl+Shift+F', FULLSCREEN );
+
+		// Add the information about the keystroke to the accessibility database.
+		this.editor.accessibility.addKeystrokeInfos( {
+			keystrokes: [
+				{
+					label: t( 'Fullscreen mode' ),
```

Comment:
There's a convention to name these entries to sound like human-friendly commands, i.e. start with a verb:

<img width="495" alt="image" src="https://github.com/user-attachments/assets/9f819e5b-0359-4477-b0a3-3c98858a57bc" />

So this should be more like "Toggle fullscreen mode".